libs/providers/openshift.py

```
@@ -153,12 +153,31 @@ def vm_dict(self, wait_for_guest_agent: bool = False, **kwargs: Any) -> dict[str
         self.start_vm(cnv_vm)
         # True guest agent is reporting all ok
         result_vm_info["guest_agent_running"] = (
             self.wait_for_cnv_vm_guest_agent(vm_dict=result_vm_info) if wait_for_guest_agent else False
         )
 
+        # Node name - where the VM is scheduled (collected after VM start)
+        node_name = None
+        try:
+            node_name = cnv_vm.vmi.instance.status.nodeName
+        except (AttributeError, NotFoundError):
+            LOGGER.debug(f"Could not retrieve node name for VM {cnv_vm_name}")
```

Ruff (0.14.11)

Logging statement uses f-string

(G004)